app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  load_and_authorize_resource
+  before_action :authenticate_user!
+  before_action :set_user, only: %i[ update destroy ]
+
+
+  def destroy    
+    @user.destroy
+    respond_to do |format|
+      format.html { redirect_to (request.referrer||admin_path),status: 302, notice: 'User was successfully deleted.' }   
+      format.json { head :no_content } 
+    end
+  end
+
+  def update
+    @user.update_column(:email, params[:user].delete(:email))
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to (request.referrer||admin_path),status: 302, notice: "User successfully updated"}
+        format.json { render :show, status: 302, location: @user }
```

Comment:
"Found" is not exactly the right status. 200 is better. Also, try to use symbols instead of integers, meaning not `status: 200`, but `status: :ok`, or `:unprocessable_entity`, or whatever.

Reply:
It more like :see_other the right status because with jquery and pagination with status :ok is not redirecting